src/Components/NavBar.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import { Link } from 'react-router-dom';
+import '../Styling/Navbar.css';
+import { MenuItems } from "./MenuItems";
+import { HiOutlineX } from "react-icons/hi";
+import SubMenu from './SubMenu';
+
+
+
+function Navbar() {
+    const [sidebar, setSidebar] = useState(false);
+
+    const showSidebar = () => setSidebar(!sidebar);
+    return(
+        <>
+            <div className="Navbar">
+                <Link to="/Home">
+                    <img className="Navbar_logo" src={require('../Assets/letter-f-2.png')} />
+                </Link>
+                <Link to="#">
+                    <img className="Menu_bars" src={require('../Assets/menu-2.png')} onClick={showSidebar}/>
```

Comment:
consider structuring you img tags like this:

`
<img 
className="Menu_bars" 
src={require('../Assets/menu-2.png')} 
onClick={showSidebar} />
`